plugins/CoreAdminHome/stylesheets/generalSettings.less

```diff
@@ -74,12 +74,31 @@ table.admin  tbody td:hover, table.admin  tbody th:hover {
     padding: 15px;
 }
 
+.admin p + h2 {
+  margin-top: 35px;
+  // .admin p defines a margin-bottom of 10px, we make sure we still have a margin-top of 45px this way
+}
+
 .admin h2 {
-    border-bottom: 1px solid #DADADA;
-    margin: 15px -15px 20px 0;
+    border-bottom: 0px;
+    margin: 45px -15px 11px 0;
     padding: 0 0 5px 0;
     font-size: 24px;
     width:100%;
+
+  &:first-of-type:not(.secondary) {
+    margin-top: 17px;
+    margin-bottom: 16px;
+    border-bottom: 1px solid #DADADA;
+  }
+}
+
+.admin h2 + h3 {
+  margin-top: 0px;
+}
+
+.admin h3 {
+  margin-top: 30px;
 }
```

Comment:
The other styles in this selector were already 4 spaces so I continued with it. It's different in every file... maybe it makes sense to run a tool over all less files to change the indentation? 

I think the admin headers are supposed to look a bit different (have different spaces and border) than the reporting UI. Maybe it some point we need to introduce 2 classes. One that identifies the reporting UI and one class that identifies all other parts of the UI